src/test/java/unbanner/StudentTest.java

```diff
@@ -13,6 +15,7 @@
   public void initObjects() {
     bob = new Student("Bob", "Jones", 0); 
     bob.setId("Dummy String Test");
+    bob.setCourses(new ArrayList<Course>());
```

Comment:
should be using either a dummy or a mock for course objects inside of this student class test.